vsphere/metadata.csv

```diff
@@ -297,6 +297,10 @@ vsphere.host.summary.runtime.powerState,count,300,,,The host power state. Tagged
 vsphere.datastore.count,gauge,,,,Timeserie with value 1 for each Datastore. Make 'sum by {X}' queries to count all the Datastores with the tag X.,0,vsphere,datastore count,
 vsphere.datacenter.count,gauge,,,,Timeserie with value 1 for each Datacenter. Make 'sum by {X}' queries to count all the Datacenters with the tag X.,0,vsphere,datacenter count,
 vsphere.cluster.count,gauge,,,,Timeserie with value 1 for each Cluster. Make 'sum by {X}' queries to count all the Clusters with the tag X.,0,vsphere,cluster count,
+vsphere.cluster.configuration.drsConfig.enabled,count,300,,,"Whether or not the VMware DRS service is enabled. Tagged by `enabled`.",0,vsphere,cluster drs enabled,
```

Comment:
Am I correct to assume the value of these metrics don't matter and that the only thing that matters is the tag?

Reply:
yes!